src/components/Main.js

```diff
@@ -57,147 +91,80 @@ class Main extends React.Component {
    */
   updatePathState(evt, {path, index = null}) {
     // put the paths actual state to the log
-    const entry = {
+    this.conversationLog.push({
       stateAtPos: JSON.stringify(this.state),
       index
-    };
-    this.conversationLog.push(entry);
+    });
     // trigger next path
     this.setState({path: path});
   }
 
   handleInputfieldEnter({evt, path, index, changeVal}) {
     /* ToDo: chrome complains about this enter detection beeing deprecated */
```

Comment:
but `event.key` as well as `event.which` are all deprecated, according to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/which). But for the quick I only found `evt.code` that doesn't seem to be working for some unknown reason. So what do you suggest?